src/components/Sender/Recipients.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  Checkbox,
+  Grid,
+  List,
+  ListItem,
+  ListItemText,
+} from '@material-ui/core';
+import React from 'react';
+import { SpreadsheetCtx } from '../../contexts/spreadsheet.context';
+
+const Recipients = () => {
+  const [spreadsheet, setSpreadsheet] = React.useContext(SpreadsheetCtx);
```

Comment:
Avoid using `React.*`, import `useContext` instead

Reply:
But why? if we have already imported `React` object isn't it make faster?